

export const getPayments = async (req,res,next) => {

    try
    {}
    catch(error)
    {
        next(error)
    }

}


export const getPayment = async (req,res,next) => {

    try
    {}
    catch(error)
    {
        next(error)
    }

}


export const deletePayment = async (req,res,next) => {

    try
    {}
    catch(error)
    {
        next(error)
    }

}